spikeinterface/widgets/figurl/timeseries.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import warnings
+
+from ..timeseries import TimeseriesWidget
+from ..utils import array_to_image
+from .base_figurl import FigurlPlotter
+
+try:
+    from figurl_tiled_image import TiledImage
+    HAVE_TILED_IMAGE = True
+except ImportError:
+    HAVE_TILED_IMAGE = False
+
+
+class TimeseriesPlotter(FigurlPlotter):
+
+    def do_plot(self, data_plot, **backend_kwargs):
+        assert HAVE_TILED_IMAGE, ("To use the figurl backend for timeseries, you neeed the 'figurl_tiled_image'. "
+                                  "Install it with >>> pip install figurl_tiled_image")
+        if "color_range" in backend_kwargs:
+            color_ranges = backend_kwargs["color_range"]
+        else:
+            color_ranges = 200
+        if not isinstance(color_ranges, list):
+            color_ranges = [color_ranges] * len(data_plot["layer_keys"])
+            
+        if "tile_size" in backend_kwargs:
+            tile_size = backend_kwargs["tile_size"]
+        else:
+            tile_size = 512
+            
```

Comment:
```suggestion
            tile_size = backend_kwargs.get("tile_size", 512)
```
I would do that but more generaly I would create a default dict that would be copied and update with backend_kwargs no ?